explode/rpm.go

```diff
@@ -49,7 +49,11 @@ func RPM(pkgs []string) (string, error) {
 			"-u",
 		}...)
 		// Pipe rpm into cpio
-		r, w := io.Pipe()
+		r, w , err:= os.Pipe()
```

Comment:
Please format this line as:
```
r, w, err := os.Pipe()
```

Reply:
ok, no problem

Reply:
By the way, I detected the style issues because my editor (vim) uses vim-go by default, and it automatically corrects them when saving a .go file :-)